styles.css

```diff
@@ -83,25 +107,73 @@ p {
   font-size: 1.8em;
   font-weight: 500;
   font-family: 'Poppins', sans-serif;
+  overflow: hidden;
+  cursor: pointer;
+  border: 0;
 }
 
 .boxes__lockbox {
   height: 3.3em;
-  margin-bottom: .5em;
   border: none; 
-  background: none; 
+  background: none;
+  padding: 0;
+  margin-bottom: -.5em; 
+  cursor: pointer;
 } 
 
 .boxes__unlocked {
-  height: 3em;
-  margin-bottom: .4em;
+  height: 2em;
 }
 
 .boxes__locked {
-  height: 3em;
-  margin-bottom: .4em;
+  height: 2em;
 }
 
 .hidden {
   display: none;
+}
+
+/* saved section */
```

Comment:
Do we need all these organizational comments?

Reply:
Let's review this in our final refactor phase.

Reply:
I don't think so. We can discuss this at our next meeting!